arwen/contexts/reservedFunctions_test.go

```diff
@@ -7,13 +7,13 @@ import (
 )
 
 func TestReservedFunctions_IsFunctionReserved(t *testing.T) {
```

Comment:
You could add another test in which you pass a new reserved function on the constructor, and see that it is actually added.

Reply:
Added extra test for this.